scripts/streaming-watcher.py

```
@@ -1093,14 +1026,18 @@ async def process_file(self, file_path: Path) -> bool:
-                }
+                # Mark file as processed with 0 chunks using UnifiedStateManager
+                try:
+                    self.state_manager.add_imported_file(
+                        file_path=str(file_path),
+                        chunks=0,
+                        importer="streaming",
+                        collection=collection_name,
+                        embedding_mode="local" if self.config.prefer_local_embeddings else "cloud",
+                        status="completed"
+                    )
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)